src/nodejs/lambda-layers/database-util/src/query/section.js

```diff
@@ -54,4 +54,31 @@ const formJoin = () => sql.select({
   alias: 'section_agg'
 });
 
+const createSection = (params) => sql.insert({
+  ...{
+    table: `section (${params.id ? 'id,': ''} form_id, heading, list_order, required_if, show_if, daac_id)`,
+    values: {
+      type: 'values_list',
+      items: [
+        ...(params.id ? ['{{id}}'] : []),
+        '{{form_id}}', '{{heading}}', '{{list_order}}',
+        params.required_if ? `'${JSON.stringify(params.required_if)}'::JSONB`: "'[]'::JSONB",
+        params.show_if ? `'${JSON.stringify(params.show_if)}'::JSONB`: "'[]'::JSONB",
+        params.daac_id? '{{daac_id}}':'null',
```

Comment:
What are your thoughts on removing the explicit default declarations? For example:
```
    table: `section (${params.id ? 'id,': ''} form_id, heading, list_order${params.required_if ? ', required_if' : ''}${params.show_id ? ', show_if' : ''}${params.daac_id ? ', daac_id)' : ''}`,
    values: {
      type: 'values_list',
      items: [
        ...(params.id ? ['{{id}}'] : []),
        '{{form_id}}', '{{heading}}', '{{list_order}}',
        ...(params.required_if ? `'${JSON.stringify(params.required_if)}'::JSONB` : []),
        ...(params.show_if ? [`'${JSON.stringify(params.show_if)}'::JSONB`] : []),
        ...(params.daac_id? '{{daac_id}}': []),
```
Mainly trying to avoid any confusions around the db defaults versus explicit defaults we add.

Reply:
I waffled on doing it originally so happy to make the change. Only reason I didn't was in someways it makes it harder to read.